pkg/ddl/executor.go

```diff
@@ -2261,18 +2261,24 @@ func (e *executor) AddColumn(ctx sessionctx.Context, ti ast.Ident, spec *ast.Alt
 	}
 
 	job := &model.Job{
+		Version:        model.JobVersion1,
```

Comment:
Why do we still use `model.JobVersion1`?

Reply:
multi schema change job types will handle together after they are all refactored

Reply:
Because the addColumn ddl will be added into multiSchemaChange ddl  subjob. The jobVersion of multiSchemaChange is V1, the jobVersion of addColumn ddl is from multiSchemaChange.